patches/minecraft/net/minecraft/entity/player/EntityPlayer.java.patch

```diff
@@ -287,6 +287,15 @@
          if (p_71059_1_.func_70075_an())
          {
              if (!p_71059_1_.func_85031_j(this))
+@@ -1180,7 +1263,7 @@
+                 {
+                     f1 = EnchantmentHelper.func_152377_a(this.func_184614_ca(), EnumCreatureAttribute.UNDEFINED);
+                 }
+-
```

Comment:
leave this newline here, do not remove it

Reply:
Why? Idea was to prevent an additional line as described in the Contribution Guidelines https://github.com/MinecraftForge/MinecraftForge/wiki/If-you-want-to-contribute-to-Forge#conventions-for-coding-patches-for-a-minecraft-class-javapatch

Reply:
that's to avoid patches, but here you are making patches.

Reply:
I'm sorry, I don't get this. Aren't you always making patches if you modify/patch Minecraft code?
Fixed anyway

Reply:
Never mind, I think I have misinterpreted the guidelines